resources/dicts/patrols/general/medcat.json

```diff
@@ -628,26 +1185,47 @@
                 "exp": 0,
                 "weight": 20
             },
+            {
+                "text": "p_l sits to greet the presence, but it vanishes, leaving the medicine cat to wonder if {PRONOUN/p_l/subject} {VERB/p_l/have/has} done something wrong.",
+                "exp": 0,
+                "weight": 20
+            },
             {
                 "text": "s_c turns to the ghost with a smile, but the StarClan cat meets {PRONOUN/s_c/poss} eyes with fire in their gaze. StarClan is unhappy with {PRONOUN/s_c/object}, and things must change.",
                 "exp": 0,
                 "weight": 20,
-                "stat_trait": ["bloodthirsty", "vengeful", "sneaky"]
+                "stat_trait": ["bloodthirsty", "vengeful", "sneaky", "cold", "arrogant"]
+            },
+            {
+                "text": "The connection wavers, and as quickly as the feeling of company appears, it vanishes. p_l let out a small sigh, gaze turning upwards, did StarClan not want to speak with {PRONOUN/p_l/object}?",
+                "exp": 0,
+                "weight": 10
             }
         ],
         "antag_fail_outcomes": [
             {
                 "text": "r_c feels hounded, never given a moment's peace, and {PRONOUN/r_c/subject} {VERB/r_c/hiss/hisses} at the StarClan cat. But when {PRONOUN/r_c/subject} {VERB/r_c/turn/turns} around, there's only a rather upset looking Clanmate, there to fetch {PRONOUN/r_c/object} to camp to help with an emergency.",
                 "exp": 0,
                 "weight": 20
+            },
+            {
+                "text": "r_c feels hounded, never given a moment's peace, and {PRONOUN/r_c/subject} {VERB/r_c/hiss/hisses} at the StarClan cat. The ghost shakes their head in dissaproval, but vanishes.",
+                "exp": 0,
+                "weight": 20,
+                "outsider_rep": -1
```

Comment:
Why would outsider rep decrease?

Reply:
I think that might have originally been there? If not I probably added it by mistake. I'll remove it.